docs/core/install/macos-notarization-issues.md

```diff
@@ -87,5 +87,5 @@ If you want your application to run on macOS Catalina (version 10.15) or higher,
 
 ## Next steps
 
-- [.NET Core dependencies and requirements](dependencies.md).
+- [.NET Core dependencies and requirements](windows.md).
```

Comment:
This article is for macOS, how can this link that is Windows-specific be useful as a next step for macOS user?

Reply:
I used a tool to replace any links that are redirected in the redirection JSON file, so it currently links there already. But, I see your point and I'll change it.